README.md

```diff
@@ -44,15 +44,16 @@ const secondWord = "squeeze";
 const thirdWord = "the";
 const fourthWord = "cheese";
 ```
-Result should be:
+R esult should be:
 ```js
 "Please squeeze the cheese"
+
 ```
 
 ### Your solution here:
 4.  Fill in the `console.log()`?
 ```js
-console.log()
+`console.log(firstWord +secondWord+ thirdWord + fourthWord)
```

Comment:
be careful using `

to add proper spacing do this `console.log(firstWord + " " + secondWord + " " + thirdWord + " " + fourthWord)`